apps/fluent-tester/src/FluentTester/TestComponents/Theme/ThemeTest.tsx

```diff
@@ -92,7 +90,7 @@ const SwatchList: React.FunctionComponent<{ hostKey: string }> = ({ hostKey }: {
   }, []);
   return (
     <View style={[commonTestStyles.view]}>
-      <Text variant="bodySemibold">getHostSettingsWin32(theme: ITheme).{hostKey}</Text>
+      <Text>getHostSettingsWin32(theme: ITheme).palette</Text>
```

Comment:
We haven't done the theming work on Apple platforms yet, but will this not eventually be a problem having a win32 specific call in a shared file?

Reply:
See the comment above, it should work. Previously the mac control versions were always routing through the win32 native module fallback code. With the changes to foundation-compose and the theming system this is no longer the case.